setup.py

```diff
@@ -86,7 +86,7 @@ def _get_dbt_core_version():
     },
     install_requires=[
         "dbt-core~={}".format(dbt_version),
-        "trino==0.321.0",
+        "trino==0.322.0",
```

Comment:
@mdesmet, should this be the following instead?
```suggestion
        "trino>=0.321.0<1.0.0",
```